testbed/scripts/cold-start-readiness.sh

```
@@ -0,0 +1,244 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly health_budget_ms=5000
+readonly flatness_budget_ms=2000
+readonly ready_budget_ms=120000
+readonly bootstrap_budget_ms=1200000
+readonly port=8080
+readonly base_url="http://[::1]:${port}"
+cabal build exe:nhtestbed >/dev/null
+readonly app_binary="$(cabal list-bin exe:nhtestbed)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)